packages/vue-components/src/Dropdown.vue

```diff
@@ -161,4 +161,14 @@ export default {
 .dropdown-toggle {
   cursor: pointer;
 }
+
+/* Fix for Navbar dropdown menu not showing in Bootswatch v4.5.1 */
+a:not([href]) {
+    color: inherit;
```

Comment:
let's forward these to `.dropdown-toggle` class instead, since we're explicitly relying on the user to place `nav-link` class at the moment